package.json

```diff
@@ -82,7 +82,12 @@
     "postcss": "^8.2.1",
     "browserslist": "^4.16.5",
     "safe-buffer": "^5.1.1",
-    "vfile-message": "^2.0.2"
+    "vfile-message": "^2.0.2",
+    "parcel/@parcel/config-default/@parcel/optimizer-cssnano/cssnano/cssnano-preset-default/postcss-svgo/svgo/css-select/css-what": "5.0.1",
```

Comment:
Are we forced to do this because we can't update parcel?

Reply:
Well we're updating parcel to v2 in https://github.com/cdr/code-server/pull/3578 but it doesn't have the fixes either 😅 AFAICT so this is the only solution.